python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+    cond_value = cond_value_list[index]
+    cond_dist = otexp.PointConditionalDistribution(dist_X, cond_indices, [cond_value])
+    draw = cond_dist.drawPDF().getDrawable(0)
+    draw.setLegend(r"$x_1 = q($" + str(q_list[index]) + ")")
+    g_cond.add(draw)
+    print(
+        "Quantile of the cond value = ", q_list[index], "Range : ", cond_dist.getRange()
+    )
+
+g_cond.setTitle(r"PDF of $X_0|X_1 = x_1$")
+g_cond.setColors(ot.Drawable.BuildDefaultPalette(len(q_list) + 1))
+
+# %%
+# We gather both graphs into one grid.
+grid = ot.GridLayout(1, 2)
+grid.setGraph(0, 0, g_X)
+grid.setGraph(0, 1, g_cond)
+view = otv.View(grid)
+
+# %%
+# Case 2: Bivariate distribution with continuous and discrete marginals
+# ---------------------------------------------------------------------
+#
+# Here, we consider the bivariate distribution of :math:`(X_0, X_1)` defined by:
+# .. math::
+#
+#   X_0 & \sim \cBin(25, 0.2)
+#   X_1 & \sim \cP(1)
+#   \mbox{copula} & \sim \mbox{GumbelCopula}(2)
+#
+# We condition the marginal :math:`X_1` to be simultaneously equal to its quantiles of order
+# 0.05, 0.5 and 0.95. Then, we draw the resulting conditioned distribution of:
+# .. math::
+#
+#    X_0 | X_1 = x_1
+#
+# We go through the same steps as before.
+coll_marg = [ot.Binomial(25, 0.2), ot.Poisson(1.0)]
+cop = ot.GumbelCopula(2)
+dist_X = ot.JointDistribution(coll_marg, cop)
+
+# %%
+# We draw the PDF of :math:`(X_0, X_1)`.
+g_X = dist_X.drawPDF()
+g_X.setTitle(r"$(X_0, X_1)$: iso-lines PDF")
+g_X.setXTitle(r"$x_0$")
+g_X.setYTitle(r"$x_1$")
+
+# %%
+# We define the conditioned marginal, the list of the conditioning values and we create
+# all the conditioned distribution, whose PDF we draw on the same graph.
+#
+# We also print the updated range of the distribution of :math:`X_0`.
+cond_value_list = [coll_marg[1].computeQuantile(q)[0] for q in q_list]
+
+g_cond = coll_marg[0].drawPDF()
+g_cond.setLegends([r"dist of $X_0$"])
+for index, cond_value in enumerate(cond_value_list):
+    cond_value = cond_value_list[index]
+    cond_dist = otexp.PointConditionalDistribution(dist_X, cond_indices, [cond_value])
+    draw = cond_dist.drawPDF().getDrawable(0)
+    draw.setLegend(r"$x_1 = q($" + str(q_list[index]) + ")")
+    g_cond.add(draw)
+    print(
+        "Quantile of the cond value = ", q_list[index], "Range : ", cond_dist.getRange()
+    )
+
+g_cond.setTitle(r"PDF of $X_0|X_1 = x_1$")
+g_cond.setColors(ot.Drawable.BuildDefaultPalette(len(q_list) + 1))
+
+# %%
+# We gather both graphs into one grid.
+grid = ot.GridLayout(1, 2)
+grid.setGraph(0, 0, g_X)
+grid.setGraph(0, 1, g_cond)
+view = otv.View(grid)
+
+
+# %%
+# Case 3: Trivariate distribution with continuous and discrete marginals
+# ----------------------------------------------------------------------
+#
+# Here, we consider the trivariate distribution of :math:`(X_0, X_1, X_2)` defined by:
+# .. math::
+#
+#   X_0 & \sim \cN(0, 1)
+#   X_1 & \sim \cP(1)
```

Comment:
There is no universal notation for each distribution: I suggest to explain what means the caligraphic P.

Reply:
yes